CHANGELOG.md

```diff
@@ -77,6 +77,15 @@ Converting from v3.x to v4.0
     - state.setTime(0.0);
     - Manager manager(model, state);
     - manager.integrate(1.0);
+- `Manager::setIntegrator(SimTK::Integrator)` has been removed and replaced by
+  `Manager::setIntegratorMethod(IntegratorMethod)` which uses an enum and
+  `Manager::setIntegratorMethod(std::string)` which uses a string. The latter
```

Comment:
The string option has been removed, correct?

Reply:
Yes, good catch. Should be addressed in latest commit.